internal/api/handlers/customer.go

```
@@ -0,0 +1,33 @@
+	return &CustomerHandler{
+		CustomerService: service.NewCustomerUseCase(CustomerRepository),
+	}
+}
+
+func (h *CustomerHandler) CreateCustomer(c echo.Context) error {
+	var customerCreateRequest usecase.CustomerInput
+	if err := c.Bind(&customerCreateRequest); err != nil {
+		return c.JSON(http.StatusBadRequest, "error binding create customer request")
+	}
+
+	h.CustomerService.CreateCustomer(&customerCreateRequest)
```

golangci-lint (1.64.8)

Error return value of `h.CustomerService.CreateCustomer` is not checked

(errcheck)